packages/twenty-front/src/modules/ui/layout/modal/components/Modal.tsx

```diff
@@ -15,9 +15,15 @@ const StyledModalDiv = styled(motion.div)<{
   size?: ModalSize;
   padding?: ModalPadding;
   isMobile: boolean;
+  modalVariant: ModalVariants;
 }>`
   display: flex;
   flex-direction: column;
+  box-shadow: ${({ theme }) => theme.boxShadow.superHeavy};
```

Comment:
You probably forgot to remove that line after implementing the variant below 🙂

Reply:
Thanks @Weiko!